README.md

```
@@ -0,0 +1,45 @@
+```shell
+cd ./frontend
+pnpm dev
+```
+
+Then on other terminal start the backend
+
+```shell
+cargo run
+```
+
+Then visit http://localhost:5173
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)